Library/Formula/rdup.rb

```diff
@@ -12,6 +12,10 @@ class Rdup < Formula
     sha256 "955fc1c20fd4d9daddef07f0cc4f8475f310040ed797ba0e00e1a76f91a74a46" => :mavericks
   end
 
+  option "without-test", "Skip build-time tests (not recommended)"
```

Comment:
Yeah I see where you're coming from, but I can't see how
```
depends_on "deja-gnu" => :build
depends_on "gnu-sed" => :build
depends_on "coreutils" => :build
depends_on "gnu-tar" => :build
ENV.prepend_path "PATH", Formula["gnu-sed"].opt_libexec/"gnubin"
ENV.prepend_path "PATH", Formula["coreutils"].opt_libexec/"gnubin"
ENV.prepend_path "PATH", Formula["gnu-tar"].opt_libexec/"gnubin"
```
could reasonably be considered better than
```
depends_on :gnu
```